docs/Migration Guide.md

```diff
@@ -3,6 +3,146 @@
 This document describes how to port your code from using one ESR version
 of SpiderMonkey to the next ESR version.
 
+## ESR 115 to ESR 128 ##
+
+### New API for column numbers ###
+
+Previously, column numbers (in source locations and stack frames) were
+stored as mostly as numbers starting from 0.
+When column numbers are shown in a user-facing context, however, they
+are usually displayed as starting at 1.
+SpiderMonkey is in the process of changing the internal representation
+of column numbers to start at 1 as well.
+There is a new set of APIs in `<js/ColumnNumber.h>` that encodes the
+origin of 0 or 1 in order to avoid confusion.
```

Comment:
At the point of 128, the types defined in `ColumnNumber.h` are only for 1-origin.
(there had been types both for 0-origin and 1-origin during the migration, but the migration is almost done for SpiderMonkey internal, and the 0-origin types are removed by [bug 1862692](https://bugzilla.mozilla.org/show_bug.cgi?id=1862692)).

Now the 0-origin value is used only as input (`fromZeroOrigin` methods), and internal representation is always 1-origin.